fabfile/hammerdb_confs/master.ini

```diff
@@ -0,0 +1,23 @@
+[DEFAULT]
+postgres_citus_version: ('12.1', 'release-9.2')
+postgresql_conf: [
+    # the following two are necessary to prevent getting timeouts, do not change them.
+    "tcp_keepalives_idle = 120",
+    "tcp_keepalives_interval = 30",
+    "autovacuum_vacuum_cost_delay = 0",
+    "checkpoint_completion_target = 0.9",
+    "checkpoint_timeout = '30min'",
+    "max_connections = 500",
+    "max_prepared_transactions = 300",
+    "min_wal_size = 32",
+    "max_wal_size = '50GB'",
```

Comment:
it is 50GB on hyperscale and max_connections are 600 on workers but still 300 on coordinator on hyperscale I guess. So maybe we can change it to 600 instead of 500 here.

Reply:
Let's keep 300 for now, because `500` on the coordinator could generate a lot of load.

Reply:
A typical test run is 1250 warehouses and 250 vuusers. If we use 300 it will fail. So should we keep it 500 then @onderkalaci ?